client/src/plugins/element-templates-modal/ElementTemplatesModal.js

```diff
@@ -60,6 +62,10 @@ export default class ElementTemplatesModal extends PureComponent {
     });
   }
 
+  handleBpmnModelerCreate = ({ modeler }) => {
+    this.setState({ elementTemplates: modeler.get('elementTemplates') });

```

Comment:
This could lead to some problems for the upcoming Zeebe diagrams support since we fire `bpmn.modeler.created` for those as well, but the `elementTemplates` feature will be unavailable.

Reply:
General: maybe we should consider only handling this case for Camunda BPMN diagrams.

Reply:
Good point, but I think we can manage that (ie. dont do anything if no elementTemplate service is there). But we should add a test case for that then.